resources/views/service/layouts/app.blade.php

```diff
@@ -13,6 +13,8 @@
     <!-- Styles -->
     <style>@import url('https://fonts.googleapis.com/css?family=Roboto');</style>
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <!-- Scripts, so jquery/bootstrap are available -->
+    <script src="{{ asset('js/app.js') }}"></script>
```

Comment:
How come this has moved to the head? Do we want the browser to wait for this to download and execute before rendering the document?

Reply:
I need it executed before I can find jquery in the page. this was the easiest way to do that.

Reply:
Ah I see, because you're using jQuery in create.blade.php